packages/elements/src/chart/helpers/legend.ts

```diff
@@ -1,86 +1,97 @@
-// import type { ChartDataSetsColor } from './types';
+import {
+  // eslint-disable-next-line import/extensions
+  ChartDataset, Color } from 'chart.js/auto';
 
-// /**
-//  * Validate color
-//  *
-//  * @param {ChartDataSetsColor} color datsets color
-//  * @returns {(string|undefined)} color
-//  */
+interface x {
+  pointBorderColor: Color;
+  pointBackgroundColor: Color;
+  borderColor: Color;
+  backgroundColor: Color;
+}
+type ChartDatasetNew = ChartDataset & x;
+/**
+ * Validate color
+ *
+ * @param {ChartDataSetsColor} color datsets color
+ * @returns {(string|undefined)} color
+ */
 // const isValidColor = (color: ChartDataSetsColor): string | undefined => {
-//   /**
-//    * Force return to string type because Chart.js doesn't support typescript
-//    * and some type of @types/chart.js isn't correct
-//    */
-//   return (color && color !== 'transparent') ? color as string : undefined;
-// };
+const isValidColor = (color: Color): Color | undefined => {
+  /**
+   * Force return to string type because Chart.js doesn't support typescript
+   * and some type of @types/chart.js isn't correct
```

Comment:
this comment and logic looks outdated. Could you double check this?

Reply:
This file has been removed due to the legend logic will use from chart.js instead.